web/client/.eslintrc.cjs

```diff
@@ -0,0 +1,74 @@
+module.exports = {
+	root: true,
+	env: {
```

Comment:
Is its normal that the style is fucked in this file?

Reply:
Fixed in new commit